app/server/src/features/accounts/resolvers.ts

```diff
@@ -67,5 +72,19 @@ export const resolvers: Resolvers = {
                 return toUserId(deletedUser);
             });
         },
+        async makeOrganizer(parent, args, ctx, info) {
+            return runMutation(async () => {
+                if (!ctx.viewer.id) throw new Error(errors.noLogin);
+                const updatedUser = await User.makeOrganizer(ctx.prisma, args.input, ctx.viewer.id);
+                return toUserId(updatedUser);
+            })
+        },
+        async removeOrganizer(parent, args, ctx, info) {
+            return runMutation(async () => {
+                if (!ctx.viewer.id) throw new Error(errors.noLogin);
```

Comment:
This is a resolver not a method though, and no details are exposed by this error. It is consistent with all the other resolver errors. Do you want all of them updated for just a no login error? I don't see why we would need a different user facing and internal message.

Reply:
You're correct. I missed changing the errors in resolvers to protected errors. Currently, this would not show to the user because in the runMutation method I made it such that no error is shown to the user UNLESS it is a protected error with a user message.

So, this should still be changed so the user can see it.